opensearch_py_ml/ml_commons/model_execute.py

```diff
@@ -0,0 +1,48 @@
+# SPDX-License-Identifier: Apache-2.0
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+import json
+
+from opensearchpy import OpenSearch
+
+from opensearch_py_ml.ml_commons.ml_common_utils import ML_BASE_URI
+
+
+class ModelExecute:
+    """
+    Class for executing algorithms using ML Commons execute API.
+    """
+
+    API_ENDPOINT = "models/_execute"
+
+    def __init__(self, os_client: OpenSearch):
+        self._client = os_client
+
+    def _execute(self, algorithm_name: str, input_json: dict) -> dict:
```

Comment:
first of all I think we should expect a string for `input_json` not a dict.

Reply:
Right now, it accepts both dict and string. Should I change anything?